Dockerfile-ubuntu

```diff
@@ -25,6 +30,13 @@ RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 RUN update-ca-certificates
 
+
+# Lock dotenv version to prevent ruby 3.0 requirement
```

Comment:
I'd apply this same comment to the `centos` one if it applies

Reply:
oh yeah it does, good call